client/my-sites/themes/theme-preview.jsx

```diff
@@ -19,37 +19,60 @@ export default React.createClass( {
 	propTypes: {
 		theme: React.PropTypes.object,
 		showPreview: React.PropTypes.bool,
+		showSecondaryButton: React.PropTypes.bool,
```

Comment:
I suppose this could be decided instead by truthy/falsy `secondaryButtonLabel` to keep the number of props down, but it is perfectly fine as it is.